content/docs/2.14/scalers/dynatrace.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Dynatrace"
+availability = "2.14+"
+maintainer = "Community"
+description = "Scale applications based on Dynatrace metric data points"
+go_file = "dynatrace_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `dynatrace` trigger that scales based on Dynatrace metric data points.
+
+```yaml
+triggers:
+  - type: dynatrace
+    metadata:
+      host: https://dummy-instance.live.dynatrace.com/
+      token: "NRAK-xxxxxxxxxxxxxxxxxxxxxxxxxxx"
```

Comment:
This should be coming from trigger authentication and no longer through inlined secrets, we no longer introduce new ones sorry!

Reply:
Sounds like a good practice to me, should it not be allowed at all or just not encouraged?

Reply:
Updated on the code side to not fetch from trigger metadata